simulariumio/trajectory_converter.py

```diff
@@ -168,38 +168,9 @@ def translate_positions(data: AgentData, translation: np.ndarray) -> AgentData:
             XYZ translation
         """
         total_steps = data.times.size
```

Comment:
call the translate filter here instead of re-implementing it? I forgot we had this redundant method

Reply:
the translate filter expects the full `TrajectoryData` object to be fully formed when it's called, which isn't quite true when we're calling this function from various converters as the `TrajectoryData` object is getting formed. But redundant code is bad, so I moved the redundant functionality to a util function that both places can access to handle that!